tools/cmake/CheckPythonPackage.cmake

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# CMake functions for checking for Python packages
+# Requires PYTHON_EXECUTABLE to be defined.  Call FindPython first!
```

Comment:
Where should `FindPython` be called? I do not see where you called it.

Reply:
FindPython3 is called in app.cmake.

Reply:
I do not see it `FindPython3` in this PR.

Reply:
Right here: https://github.com/multiplemonomials/mbed-os/blob/abd38b9fc72567ffef7ddea218ceba7bd40b6d0c/tools/cmake/app.cmake#L24